src/org/jitsi/impl/neomedia/transform/fec/FECTransformEngine.java

```diff
@@ -144,25 +144,32 @@ public FECTransformEngine()
 
         // Assumption: all packets in pkts have the same SSRC
         Long ssrc = findSSRC(pkts);
-        if (ssrc == null)
+        long primarySsrc = getPrimarySsrc(ssrc);
+        if (primarySsrc == -1)
         {
             return pkts;
         }
 
         AbstractFECReceiver fpt;
```

Comment:
Can we rename "fpt" to something more descriptive?

Reply:
fixed